### ISSUE TITLE
Error with recent bottle versions

### ISSUE THREAD

**Issue by eht16** [romanvm/python-web-pdb#73]
When using web-pdb with bottle version >= 0.13, web-pdb will crash:
```
CRITICAL:root:Web-PDB: starting web-server on http://127.0.0.1:5555
Exception in thread Thread-1 (_run_server):
Traceback (most recent call last):
  File "/usr/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/tmp/venv/lib/python3.12/site-packages/web_pdb/web_console.py", line 126, in _run_server
    app.frame_data = self._frame_data
    ^^^^^^^^^^^^^^
  File "/tmp/venv/lib/python3.12/site-packages/bottle.py", line 1143, in __setattr__
    raise AttributeError("Attribute %s already defined. Plugin conflict?" % name)
AttributeError: Attribute frame_data already defined. Plugin conflict?
```

I'm not sure if this is an issue in web-pdb or bottle though it seems to me the related code in https://github.com/bottlepy/bottle/blob/master/bottle.py#L1124 is pretty much on purpose.

On bottle version before 0.13 everything works fine.

**Comment by Roosenn** [comment]
I see the same issue and have the same experience in terms of old version working and new throwing this error.

**Comment by romanvm** [comment]
Thanks for reporting this. Strangely enough, it does not happen in tests, and the line in question in the Bottle sources added long time ago.

**Comment by 13290207568** [comment]
I met the same problem. I use python 3.8.20, web-pdb 1.6.2 and the bug disappeard when I changed bottle to 0.12.25
```
CRITICAL:root:Web-PDB: starting web-server on http://14pro:5555
Exception in thread Thread-1:
Traceback (most recent call last):
  File "C:\Users\xxx\.conda\envs\xxx\lib\threading.py", line 932, in _bootstrap_inner
    self.run()
  File "C:\Users\xxx\.conda\envs\xxx\lib\threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "c:\users\xxx\.conda\envs\xxx\lib\site-packages\web_pdb\web_console.py", line 126, in _run_server
    app.frame_data = self._frame_data
  File "c:\users\xxx\.conda\envs\xxx\lib\site-packages\bottle.py", line 1142, in __setattr__
    raise AttributeError("Attribute %s already defined. Plugin conflict?" % name)
AttributeError: Attribute frame_data already defined. Plugin conflict?
```

**Comment by Dan-Eli** [comment]
I also met the same issue. I use Python 3.11.0 and web_pdb 1.6.2 
Is installing a lower bottle version will correct the problem (lower than 0.13)?

**Comment by romanvm** [comment]
Fixed in 1.6.3
